bridge.go

```diff
@@ -130,6 +137,60 @@ func (b *Bridge) CreateUserContext(ctx context.Context, n string) (string, error
 
 }
 
+// CreateUser2 creates a user by adding deviceType to the list of whitelisted users on the bridge.
+// The link button on the bridge must have been pressed before calling CreateUser.
+func (b *Bridge) CreateUser2(deviceType string, generateClientKey bool) (*CreatedUser, error) {
```

Comment:
Not a very descriptive function name. I suggest using `CreateUserWithClientKey` to make it specific to the functions implementation. And drop `generateClientKey`.

Reply:
Yeah that sounds good with me. I can make those changes.